docs/kyma-environment-broker/03-01-runtime-components.md

```diff
@@ -0,0 +1,24 @@
+# Runtime components
+
+The diagram and steps describe the Kyma Environment Broker (KEB) workflow for calculating and processing runtime components:
```

Comment:
If we decide to merge these two docs into one, we will need to rephrase the introduction.